spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -83,4 +81,46 @@
       end
     end
   end
+
+  describe 'POST #create' do
+    context 'given an invalid file' do
```

Comment:
We test happy path cases first, then add unhappy cases
```suggestion
    context 'given a valid file' do
```

Reply:
Yes, makes total sense! Thanks! :)